src/WatchDogs.Application/Program.cs

```diff
@@ -1,59 +1,76 @@
-
-
-
 using Contracts;
 using Infrastructure.DxTrade;
+using Serilog;
 using System.Net.Http.Headers;
 
-var builder = WebApplication.CreateBuilder(args);
+Log.Logger = new LoggerConfiguration()
+    .WriteTo.Console()
+    .CreateLogger();
 
-// Add services to the container.
-builder.Services.Configure<DxTradeConnectionOptions>(
-    builder.Configuration.GetSection(nameof(DxTradeConnectionOptions)));
+try
+{
+    Log.Information("Starting web application");
```

Comment:
Please move this just before `app.Run()` and add a timestamp when the application is starting.